dev/com.ibm.websphere.appserver.features/visibility/public/websocket-2.0/com.ibm.websphere.appserver.websocket-2.0.feature

```diff
@@ -0,0 +1,20 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
Changed to io.openliberty.websocket-2.0.feature

Reply:
I'll also have to address this in servlet, el, & jsp.

Reply:
servlet will NOT be updated to io.openliberty prefix.  It is special because we need to support tolerates for servlet through 5.0.  el and jsp though can be updated yes.

Reply:
Noted: https://github.com/OpenLiberty/open-liberty/issues/12916